pkg/adapter/containers_remote.go

```diff
@@ -148,6 +150,36 @@ func (r *LocalRuntime) StopContainers(ctx context.Context, cli *cliconfig.StopVa
 	return ok, failures, nil
 }
 
+// KillContainers sends signal to container(s) based on CLI inputs.
+// Returns list of successful id(s), map of failed id(s) + error, or error not from container
+func (r *LocalRuntime) KillContainers(ctx context.Context, cli *cliconfig.KillValues) ([]string, map[string]error, error) {
+	var (
+		ok       = []string{}
+		failures = map[string]error{}
+	)
+
+	killSignal := int64(syscall.SIGTERM)
+	if cli.Flags().Changed("signal") {
+		sysSignal, _ := signal.ParseSignal(cli.Signal)
```

Comment:
Why aren't we checking failure?

No need to check if Signal .Changed, always parse.

Reply:
Error already checked upstream.  Can add it here